tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -148,34 +148,39 @@ Dependencies section
 In this section you set the paths of the GDExtension dependencies. This is used internally to export the dependencies
 when exporting your game executable. You are able to set which dependency is loaded depending on the feature flags
 of the exported executable. In addition, you are able to set an optional subdirectory to move your dependencies into.
-In the following example the libraries are moved into a subdirectory called ``libraries``.
+If no path is supplied Godot will move the libraries into the same directory as your game executable.
+
+.. note::
+    In MacOS it is common to have shared libraries inside a folder called ``Frameworks`` with a directory structure
+    like this: ``Game.app/Contents/Frameworks``. As long as the libraries are located inside the ``Contents`` directory
+    Godot will be able to load them into your game executable.
```

Comment:
Are you sure that the libraries can be anywhere under `Contents`? Or, do they need to be in exactly `Contents/Frameworks`? I'm not familiar enough with MacOS.

Reply:
They can be in any folder inside the `Contents` folder. They can even be just directly inside and are picked up correctly. In XCode for instance you have differently named folders not just `Frameworks` but also `Library` where you can also find `.framework` folders and their respective libraries.